### PR TITLE
Only append default sourceMappingURL if no output wrapper is specified

### PR DESCRIPTION
**Pull request by sirajrauff**
See #2113.

### CODE REVIEW

**Comment by sirajrauff** [comment]
Note that this change does not eliminate the default sourceMapURL from being specified, which is still divergent from the behaviour when run from the command line. Without information on why this was the original behaviour, I opted to simply change it so that the default was in in lieu of a given output wrapper argument.

**Comment by ChadKillingsworth** [comment]
> Without information on why this was the original behaviour, I opted to simply change it so that the default was in in lieu of a given output wrapper argument.

I'm not sure anyone even knows that now. The ant plugin is completely community maintained.

File: src/com/google/javascript/jscomp/ant/CompileTask.java [hunk 1]
```diff
@@ -410,12 +410,13 @@ public void execute() {
             throw new BuildException("Invalid output_wrapper specified. " +
                 "Missing '" + CommandLineRunner.OUTPUT_MARKER + "'.");
           }
+        } else {
```
Comment (ChadKillingsworth) [inline src/com/google/javascript/jscomp/ant/CompileTask.java]: Do you need to check here that `result.sourceMap != null`?

**Comment by sirajrauff** [comment]
With your previous comment in mind, I removed the default output wrapper. This reduces complexity of having to check for the sourceMap or a user-given output wrapper before inserting a default, and brings this in line with the command-line behaviour.

**Comment by ChadKillingsworth** [comment]
Toggling for Travis

**Comment by brad4d** [comment]
Sorry about the delay on this.
@sirajrauff if you'll squash your branch down to 1 commit, I'll import this pull request.

**Comment by sirajrauff** [comment]
@brad4d, I've rebased of current master and squashed, let me know if there's anything else.

**Comment by brad4d** [comment]
@sirajrauff thanks
I've imported this and I'm confirming it doesn't break anything for us.

**Comment by brad4d** [comment]
No problems found by standard tests or test for example projects.
Sent for internal code review.

**Pull request closed without merging**